src/manifest/mod.rs

```diff
@@ -343,3 +343,275 @@ pub mod provenance {
         const AUTHENTICATED: bool = false;
     }
 }
+
+wire_enum! {
+    /// A policy for responding to verification failure in a read-write region.
+    ///
+    /// Cerberus currently does not fully specify what these policies mean
+    /// precisely, nor what failure mode they should be enacted with respect
+    /// to.
+    pub enum RwFailurePolicy: u8 {
+        /// Do nothing.
+        DoNothing = 0b00,
+        /// Restore the region from write-only memory.
+        RestoreFromRo = 0b01,
+        /// Erase the region.
+        Erase = 0b10,
+    }
+}
+
+impl RwFailurePolicy {
+    /// Returns the `RwFailurePolicy` flag based on a `u8`
+    pub fn from_u8(n: u8) -> RwFailurePolicy {
+        match n & 0b11 {
+            0b01 => RwFailurePolicy::RestoreFromRo,
+            0b10 => RwFailurePolicy::Erase,
+            _ => RwFailurePolicy::DoNothing,
```

Comment:
Do we want to maybe return Option here to capture that this can be out-of-bounds? I don't like silently dropping it...

Reply:
yeah, I thought about that as well. The spec says "reserved (do nothing)".

Would you want me to just add that as another variant so we are fully expressing the variants?

Also, we could have a `fn do_nothing(&self) -> bool` which would be true for `0b00` and `0b11`. 

Thoughts?